src/Tools/Source/CompilerGeneratorTools/Source/CSharpSyntaxGenerator/TestWriter.cs

```diff
@@ -263,12 +263,16 @@ private void WriteFactoryPropertyTest(Node node, bool isGreen)
                     }
                     else
                     {
-                        if (field.Type == "SyntaxToken" ||
+                        if (field.Type == "SyntaxToken")
+                        {
+                            WriteLine("Assert.NotEqual(default, node.{0});", field.Name);
+                        }
+                        else if (
                             field.Type == "SyntaxTokenList" ||
                             field.Type.StartsWith("SyntaxList<") ||
                             field.Type.StartsWith("SeparatedSyntaxList<"))
                         {
-                            WriteLine("Assert.NotEqual(default, node.{0});", field.Name);
+                            WriteLine("Assert.Equal(default, node.{0});", field.Name);
```

Comment:
💡 Consider using `Assert.Empty` here

Reply:
That would cause an unnecessary boxing  operation.

Reply:
We care about boxing operations in tests? :smile:

Reply:
> That would cause an unnecessary boxing operation.

It would also cause an unnecessary clarification of the assertion. 😆

Reply:
> We care about boxing operations in tests?

Yes-ish. Several times I've found allocations to be at the heart of unit test performance. At least in personal projects. Got me in the habit of avoiding them when I see them.